lib/matplotlib/collections.py

```diff
@@ -27,54 +27,41 @@
     "linewidth": ["linewidths", "lw"],
 })
 class Collection(artist.Artist, cm.ScalarMappable):
+    r"""
+    Base class for Collections. Must be subclassed to be usable.
+
+    A Collection represents a sequence of `~.patches.Patch`\s that can be drawn
```

Comment:
```suggestion
    A Collection represents a sequence of `.Patch`\s that can be drawn
```
and similar occurrences. (We only have one Patch class, so there is no need for disambiguation by prefixing the module.)